k8s/k8s_descriptor-poz-dev.yaml

```diff
@@ -0,0 +1,207 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: vignette
+  name: vignette
+  namespace: dev
+spec:
+  ports:
+  - port: 80
+    name: main
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: vignette
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: vignette
+    stage: deployed
+  name: vignette
+  namespace: dev
+spec:
+  selector:
+    matchLabels:
+      app: vignette
+      stage: deployed
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 2
+  progressDeadlineSeconds: 120
+  template:
+    metadata:
+      labels:
+        app: vignette
+        stage: deployed
+    spec:
+      serviceAccountName: k8s-pod
```

Comment:
Shouldn't it be `k8s-pod-dev`?

Reply:
I've changed it back to `k8s-pod` because we were getting the following error: `error looking up service account dev/k8s-pod-dev: serviceaccount "k8s-pod-dev" not found`